exchange/xgrid.F90

```diff
@@ -2257,7 +2254,7 @@ end subroutine set_comm_get1_repro
 
 !#######################################################################
 subroutine set_comm_get1(xmap)
-  type (xmap_type), intent(inout) :: xmap
+  type (xmap_type), intent(inout), target :: xmap
   type (grid_type), pointer, save :: grid1 =>NULL()
```

Comment:
guess we can't do much about these saved pointers right? might be less memory to save them as pointers anyway

Reply:
Pointers which are used as pointers should not be upgraded.